webapp/src/components/sidebar/sidebar.scss

```diff
@@ -127,4 +127,10 @@
         stroke: rgba(var(--sidebar-text-rgb), 0.5);
         stroke-width: 6px;
     }
+
+    @media screen and (min-width: 768px) {
+        .sidebarSwitcher {
+            display: none;
+        }
+    }
```

Comment:
@harshilsharma63 why is this needed? This is a kind of regression for personal server because now it is not possible to hide the side bar on wide screens.

Reply:
@kamre That's a UX decision the team took - to only allow hiding the sidebar when the window size is small.

DO you think it's something that should be supported?

Reply:
Yes, in my opinion there are situations when the space is not enough even on wide screens.

Look for example at these kanban boards from default templates, they all have horizontal scroll:
![BoardsScroll](https://user-images.githubusercontent.com/5587620/132796112-71ba9899-2351-4699-a541-484262f01a15.gif)
This is on laptop with 150% scale in Windows settings.

Reply:
cc @asaadmahmood

Reply:
FYI @harshilsharma63, sent PR #1219 to you for review to only hide the collapse-sidebar button for workspaces.